Firestore/Source/Public/FirebaseFirestore/FIRAggregateQuery.h

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Add `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` to ALL of the new header files added in this directory. Then, remove all of the `_Nonnull` annotations. See other .h files in this directory for examples.

Reply:
Done.